mapstory/static/style/themes/blue/brandcolors.less

```diff
@@ -1,8 +1,10 @@
 // === BRANDED THEME VARIABLES GO HERE ====
-@primary: #162f43; 
+@primary: #162f43;
 @primary-highlight: lighten(@primary, 10%);
 @primary-shadow: darken(@primary, 10%);
 
 @secondary: #162f43;
 @secondary-highlight: lighten(@secondary, 10%);
-@secondary-shadow: darken(@secondary, 10%);
\ No newline at end of file
+@secondary-shadow: darken(@secondary, 10%);
+
+@navigation-gradient: linear-gradient(107deg, #151617 20%, #1d5aa7 140%);
```

Comment:
Each theme has a different mix of colors for the nav gradient in its `brandcolors.less`